sw/host/opentitanlib/src/image/manifest_def.rs

```diff
@@ -96,7 +96,12 @@ macro_rules! manifest_def {
 
 impl ManifestSpec {
     pub fn read_from_file(path: &Path) -> Result<ManifestSpec> {
-        Ok(deser_hjson::from_str(&std::fs::read_to_string(path)?)?)
+        Ok(deser_hjson::from_str(
```

Comment:
This kind of error information is better communicated by using the [`anyhow::Context`](https://docs.rs/anyhow/1.0.87/anyhow/trait.Context.html) trait:
```
use anyhow::Context;
...

  std::fs::read_to_string(path).with_context(|| format!("Failed to open {path:?}"))?
```

In the error case, this will cause the error message printed by `main` to look something like:

```
Error:
    Failed to open 'foo.json'
Caused by:
    File not found
```

Reply:
done